MetaTools/generate_glyphdata.py

```diff
@@ -242,11 +112,14 @@ def generate_python_source(data, out):
         out.write('\t"%s": %s,\n' % (glyphName, glyphsCat))
     out.write("}\n\n")
 
+    out.write(
+        "DEFAULT_GLYPH_DICT = %s\n\n" % glyphs
```

Comment:
Hmmm. I believe this PR was merged to hastily.. 

We are basically now storing the GlyphData twice in `glyphdata_generated.py`: once, as the previously existing dictionaries (`PRODUCTION_NAMES`, `IRREGULAR_UNICODE_STRINGS`, etc.) that only have the stuff that cannot be derived from unicodedata, and then again, as a huge dictionary ( `DEFAULT_GLYPH_DICT`) that basically contains the whole GlyphData.xml, as it was read in by ElementTree.

Does that mean @brawer effectively gave up on the idea of storing only the delta, and is now happy with storing the full GlyphData as is? If that's the case, then I think it'd be better (ie. easier to maintain, not much slower to parse) if we just included the original xml file as a package data (possibly fetching it via git submodule), instead of writing that out as a dictionary to a python file.

Also, I don't like that the functions that parse/build a GlyphData are in that residual `util.py` module, instead of where one would expect to find them, ie. the `glyphsLib.glyphdata` module.